sitemap/sitemap.go

```diff
@@ -153,6 +156,13 @@ func addServices(baseUrl string, sitemapUrls []URL) []URL {
 
 	return sitemapUrls
 }
+func addMobileAppDevelopment(baseUrl string, sitemapUrls []URL) []URL {
+	mobileAppUrl := baseUrl + "/mobile-app-development"
```

Comment:
Let's create logic in a way that all services page can be added here. When we want to deploy a new service page, we need to only add name of that page